script/tasks/diamondSyncSigs_FAST.sh

```
@@ -0,0 +1,191 @@
+  fi
+
+  # create array with network/s for which the script should be executed
+  if [[ "$NETWORK" == "All (non-excluded) Networks" ]]; then
+    # get array with all network names
+    mapfile -t NETWORKS < <(getIncludedNetworksArray)
+  else
+    NETWORKS=("$NETWORK")
+  fi
+
+  # Determine file suffix based on environment
+  FILE_SUFFIX=$(getFileSuffix "$ENVIRONMENT")
```

Shellcheck (0.10.0)

[warning] FILE_SUFFIX appears unused. Verify use (or export if used externally).

(SC2034)